src/snippet/index.spec.jsx

```diff
@@ -42,4 +42,16 @@ two`
     expect(wrapper.html()).toEqual(paragraphs);
   });
 
```

Comment:
this test added is not checking code changes. 

`return Failed to lookup content snippet: ${children}`

Reply:
as above, will undo that line - cheers